feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/widget/SessionBottomAppBarBehavior.kt

```diff
@@ -0,0 +1,57 @@
+package io.github.droidkaigi.confsched2019.session.ui.widget
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.MotionEvent
+import androidx.coordinatorlayout.widget.CoordinatorLayout
+import androidx.core.view.children
+import androidx.core.widget.NestedScrollView
+import com.google.android.material.bottomappbar.BottomAppBar
+
+class SessionBottomAppBarBehavior(
+    context: Context? = null,
+    attrs: AttributeSet? = null
+) : BottomAppBar.Behavior(context, attrs) {
+
+    private var state: Int = STATE_SCROLLED_UP
+
+    override fun onInterceptTouchEvent(
+        parent: CoordinatorLayout,
+        child: BottomAppBar,
+        ev: MotionEvent
+    ): Boolean {
+        if (child.isShown.not() || ev.action != MotionEvent.ACTION_DOWN) {
```

Comment:
We can discuss in the issue. But Currently, We are using `!` instead of `.not()` 🙏

Reply:
I see! Thanks for your review.

Reply:
fixed! 5c9b771